app/controllers/admin/trips_controller.rb

```diff
@@ -0,0 +1,9 @@
+module Admin
+  class StationsController < BaseController
```

Comment:
Was this a copy and paste here and we need to change it to TripsController inside this?